submissions/migrations/0004_remove_django_extensions.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('submissions', '0003_submission_status'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='submission',
+            name='uuid',
+            field=models.UUIDField(default=uuid.uuid4, db_index=True),
+        ),
+    ]
```

Comment:
This is a good question - the only change is a semantic one around the `uuid` field (it has no practical effect on normal usage), but the output of `manage.py sqlmigrate` is not something I'd expect:
```
BEGIN;
ALTER TABLE `submissions_submission` ALTER COLUMN `uuid` SET DEFAULT '240af22377d44f82817ee79b1a5fd61d';
ALTER TABLE `submissions_submission` ALTER COLUMN `uuid` DROP DEFAULT;

COMMIT;
```

The size of the table is 1.7M rows in prod. However, I would argue that the above is effectively a no-op, and we ought to just skip applying the migration entirely.

@jibsheet, any suggestions here? I'm attempting to remove the `django_extensions` requirement on this project, and it's used in a model field. That change necessitated a migration, which when poked seems to do nothing, effectively. Can we declare this "safe to skip" and include a `--fake` in the GoCD rollout of this code change? Is that even possible?

If this turns out to be too complex, I'm fine with reverting the django_extensions removal out.